File: examples/otp/generate/node/generateOtp.js
import * as otp from "m3o/otp";

// Generate an OTP (one time pass) code
async function GenerateOtp() {
  let otpService = new otp.OtpService(process.env.MICRO_API_TOKEN);
  let rsp = await otpService.generate({
    id: "asim@example.com",
  });
  console.log(rsp);
}

await GenerateOtp();

File: examples/go/otp/generate/generateOtp.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/otp"
)

// Generate an OTP (one time pass) code
func GenerateOtp() {
	otpService := otp.NewOtpService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := otpService.Generate(&otp.GenerateRequest{
		Id: "asim@example.com",
	})
	fmt.Println(rsp, err)
}
